Extensions/Effects/crt-pixi-filter.js

```diff
@@ -1,13 +1,17 @@
 gdjs.PixiFiltersTools.registerFilterCreator('CRT', {
   makePIXIFilter: function(layer, effectData) {
     var crtFilter = new PIXI.filters.CRTFilter();
-
+    crtFilter._animationTimer = 0;
     return crtFilter;
   },
   update: function(filter, layer) {
-    if (filter.animated) {
-      filter.time += layer.getElapsedTime() / 1000;
-      filter.seed = Math.random();
+    if (filter.animationFrequency !== 0) { 
+      filter._animationTimer += layer.getElapsedTime() / 1000;
+      if (filter._animationTimer >= 1 / filter.animationFrequency) {
+        filter.seed = Math.random();
+        filter.time += layer.getElapsedTime() / 1000;
```

Comment:
time must be updated continuously. this is not linked to the frequency, which is only for discrete changes like seed.

Reply:
Thank you! I have some misunderstandings about `time` before and I have fixed it.